content/includes/header.html

```diff
@@ -141,4 +148,21 @@ <h1 role="banner" class="va-header-logo"><a href="/">Vets.gov</a></h1>
   var script = document.createElement("script");
   script.src = "https://search.usa.gov/javascripts/remote.loader.js";
   document.getElementsByTagName("head")[0].appendChild(script);
+
+  (function() {
+    var domReady = function(callback) {
+      document.readyState === "interactive" || document.readyState === "complete" ? callback() : document.addEventListener("DOMContentLoaded", callback);
+    };
+
+    domReady(function() {
+      // check whether browser is IE10 and older
+      if (window.navigator.userAgent.indexOf('MSIE ') > 0) {
```

Comment:
MSIE is only present in versions 10 and below; this works because our minimum version is IE11